packages/libllvm/build.sh

```diff
@@ -1,23 +1,19 @@
 TERMUX_PKG_HOMEPAGE=https://clang.llvm.org/
 TERMUX_PKG_DESCRIPTION="Modular compiler and toolchain technologies library"
 TERMUX_PKG_LICENSE="NCSA"
-TERMUX_PKG_VERSION=9.0.1
-TERMUX_PKG_REVISION=4
-TERMUX_PKG_SHA256=(00a1ee1f389f81e9979f3a640a01c431b3021de0d42278f6508391a2f0b81c9a
-		   5778512b2e065c204010f88777d44b95250671103e434f9dc7363ab2e3804253		   
-		   86262bad3e2fd784ba8c5e2158d7aa36f12b85f2515e95bc81d65d75bb9b0c82
-		   5c94060f846f965698574d9ce22975c0e9f04c9b14088c3af5f03870af75cace)
+TERMUX_PKG_VERSION=10.0.0
+TERMUX_PKG_SHA256=(df83a44b3a9a71029049ec101fb0077ecbbdf5fe41e395215025779099a98fdf
+		   885b062b00e903df72631c5f98b9579ed1ed2790f74e5646b4234fa084eacb21
+		   b9a0d7c576eeef05bc06d6e954938a01c5396cee1d1e985891e0b1cf16e3d708
+		   3b9ff29a45d0509a1e9667a0feb43538ef402ea8cfc7df3758a01f20df08adfa)
 TERMUX_PKG_SRCURL=(https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/llvm-$TERMUX_PKG_VERSION.src.tar.xz
                    https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/clang-$TERMUX_PKG_VERSION.src.tar.xz
                    https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/lld-$TERMUX_PKG_VERSION.src.tar.xz
                    https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/openmp-$TERMUX_PKG_VERSION.src.tar.xz)
 TERMUX_PKG_HOSTBUILD=true
 TERMUX_PKG_RM_AFTER_INSTALL="
-bin/clang-check
 bin/clang-import-test
-bin/clang-offload-bundler
-bin/git-clang-format
-bin/macho-dump
+bin/clang-offload-wrapper
```

Comment:
These clang binaries might be useful, while `macho-dump` isn't built anymore and the last wrapper is new and isn't in the clang package for Arch linux, so left it out.